yougov/pinot-client-rust
# DRAFT: Add primative arrays support

Add deserialisation for primitive array dimension columns.

Goals:

* Add primitive arrays to test DB
* Add deserialisation of primitive arrays in result table
* Add tests for result table for newly added types
* Update tests against test DB to check for primitive arrays
